kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/compare.kt

```diff
@@ -98,12 +108,29 @@ enum class PropertyOrder {
    /**
     * Default. Property order in objects does not matter.
     *
-    * Example: `"""{ "a": 0, "b": 2 }""".shouldEqualJson("""{ "b": 2, "a": 1 }""", compareJsonOptions { propertyOrder = Lenient })` will pass
```

Comment:
I think the original has a mistake. `a=0` on the left, but `a=1` on right. I updated it so that `a=1`.